src/js/locale/i18n.tsx

```diff
@@ -0,0 +1,17 @@
+import i18n from 'i18next';
+import { initReactI18next } from 'react-i18next';
+import en from './en.json';
+
+const translation = i18n.createInstance();
```

Comment:
I don't think we need this line because the default export is already an instance. The docs say:
> The default export of the i18next module is an i18next instance ready to be initialized by calling init.

https://www.i18next.com/overview/api#init

Reply:
I seem to remember this needs to be done because the platform already has an instance and i believe without this it replaces the platform instance?